sdk/servicebus/service-bus/src/session/sessionManager.ts

```diff
@@ -223,15 +214,14 @@ export class SessionManager {
    * @throws MessagingError if any error occurs while receiving messages from the service.
    */
   async manageMessageSessions(
-    entityType: SessionEntityType,
     onMessage: OnMessage,
     onError: OnError,
     options?: SessionManagerOptions
   ): Promise<void> {
     if (this._isManagingSessions) {
       throw new Error(
-        `${entityType}Client for "${this._context.namespace.config.entityPath}" ` +
-          `is already receiving messages from sessions. Please close this ${entityType}Client or ` +
+        `Client for "${this._context.namespace.config.entityPath}" ` +
```

Comment:
Should this be receiver and not client?
(Last I checked we are not using `manageMessageSessions` method anywhere in src/tests)

Reply:
It should now. This is the "unpublished" round-robin session client.